[2024-05-23T16:18:17.040+0000] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2024-05-23T16:18:17.052+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T16:18:14.241262+00:00 [queued]>
[2024-05-23T16:18:17.057+0000] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: games.create_db manual__2024-05-23T16:18:14.241262+00:00 [queued]>
[2024-05-23T16:18:17.057+0000] {taskinstance.py:2303} INFO - Starting attempt 1 of 1
[2024-05-23T16:18:17.068+0000] {taskinstance.py:2327} INFO - Executing <Task(PythonOperator): create_db> on 2024-05-23 16:18:14.241262+00:00
[2024-05-23T16:18:17.078+0000] {warnings.py:110} WARNING - /home/***/.local/lib/python3.12/site-packages/***/task/task_runner/standard_task_runner.py:61: DeprecationWarning: This process (pid=70) is multi-threaded, use of fork() may lead to deadlocks in the child.
  pid = os.fork()

[2024-05-23T16:18:17.079+0000] {standard_task_runner.py:63} INFO - Started process 74 to run task
[2024-05-23T16:18:17.081+0000] {standard_task_runner.py:90} INFO - Running: ['***', 'tasks', 'run', 'games', 'create_db', 'manual__2024-05-23T16:18:14.241262+00:00', '--job-id', '702', '--raw', '--subdir', 'DAGS_FOLDER/get_main_games.py', '--cfg-path', '/tmp/tmpexr575va']
[2024-05-23T16:18:17.082+0000] {standard_task_runner.py:91} INFO - Job 702: Subtask create_db
[2024-05-23T16:18:17.110+0000] {task_command.py:426} INFO - Running <TaskInstance: games.create_db manual__2024-05-23T16:18:14.241262+00:00 [running]> on host 81053f0a1c9a
[2024-05-23T16:18:17.167+0000] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='***' AIRFLOW_CTX_DAG_ID='games' AIRFLOW_CTX_TASK_ID='create_db' AIRFLOW_CTX_EXECUTION_DATE='2024-05-23T16:18:14.241262+00:00' AIRFLOW_CTX_TRY_NUMBER='1' AIRFLOW_CTX_DAG_RUN_ID='manual__2024-05-23T16:18:14.241262+00:00'
[2024-05-23T16:18:17.167+0000] {taskinstance.py:430} INFO - ::endgroup::
[2024-05-23T16:18:17.187+0000] {base.py:84} INFO - Using connection ID '***' for task execution.
[2024-05-23T16:18:18.881+0000] {logging_mixin.py:188} INFO - Таблица 'games' удалена.
[2024-05-23T16:18:19.292+0000] {python.py:237} INFO - Done. Returned value was: None
[2024-05-23T16:18:19.292+0000] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2024-05-23T16:18:19.299+0000] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=games, task_id=create_db, execution_date=20240523T161814, start_date=20240523T161817, end_date=20240523T161819
[2024-05-23T16:18:19.336+0000] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2024-05-23T16:18:19.353+0000] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2024-05-23T16:18:19.354+0000] {local_task_job_runner.py:222} INFO - ::endgroup::
